src/app/modules/bounce/bounce.controller.ts

```diff
@@ -14,9 +14,9 @@ const logger = createLoggerWithLabel(module)
  * @param res - Express response object
  */
 const handleSns = async (
-  req: Request<{}, {}, ISnsNotification>,
+  req: Request<Record<string, string>, string, ISnsNotification>,
```

Comment:
why is `ResBody` a string?

Reply:
That determines the accepted types of `res.send()` for some reason, so if this route only returns a string, that's what you set the return type to be.

All the new errors result from applying recommended typescript lint rules :P; `{}` is not recommended since it means `any non-null value` apparently.